openfisca_uk/variables/consumption/property.py

```diff
@@ -43,9 +43,9 @@ class main_residential_property_purchased(Variable):
     unit = "currency-GBP"
 
     def formula(household, period):
-        return household("property_purchased", period) * household(
-            "main_residence_value", period
-        )
+        property_purchased = household("property_purchased", period)
```

Comment:
Have thought about this - while this is equivalent to before, I think assigning to variables like this to avoid the auto-formatted brackets mess makes the formulas more readable - will do this in future, thanks.